docs/EN_US/ContainerizedHPCC/ContainerizedMods/TerraformDeployChap.xml

```diff
@@ -0,0 +1,568 @@
+
+      <sect3>
+        <title>Cloning the Terraform Repository</title>
+
+        <para>You will want to clone the Terraform Repository,
+        terraform-azurerm-hpcc hosted on the HPCC Systems GitHub
+        account.</para>
+
+        <para><ulink
+        url="https://github.com/hpcc-systems/terraform-azurerm-hpcc.git ">https://github.com/hpcc-systems/terraform-azurerm-hpcc.git</ulink></para>
+
+        <para>To clone the repository:</para>
+
+        <para><orderedlist>
+            <listitem>
+              <para>Open your command line or terminal</para>
+            </listitem>
+
+            <listitem>
+              <para>Determine where to store this project. Choose a location
+              that is easy to remember, such as the Desktop, for example:
+              Desktop/cloud/azure/terraform/ </para>
+            </listitem>
+
+            <listitem>
+              <para>Change directory to your chosen location.</para>
+            </listitem>
+
+            <listitem>
+              <para>Run  the following command :</para>
+
+              <programlisting>git clone https://github.com/hpcc-systems/terraform-azurerm-hpcc.git </programlisting>
+            </listitem>
+          </orderedlist>Once the repository is cloned, you will traverse into
+        each module's directory, and configure/modify the admin.tfvars file
+        there, and then deploy it.</para>
+      </sect3>
+
+      <sect3 id="T4MModules2Modify">
+        <title>Modules to Modify</title>
+
+        <para>The order of deployment for these Terraform modules is in fact
+        important.</para>
+
+        <para>The order that must be followed:</para>
+
+        <para><orderedlist>
+            <listitem>
+              <para>Virtual network</para>
+            </listitem>
+
+            <listitem>
+              <para>Storage account </para>
+            </listitem>
+
+            <listitem>
+              <para>Root module (AKS)</para>
+            </listitem>
+          </orderedlist></para>
+
+        <informaltable>
+          <tgroup cols="2">
+            <colspec align="left" />
+
+            <colspec align="left" />
+
+            <thead>
+              <row>
+                <entry>Modules </entry>
+
+                <entry>Location </entry>
+              </row>
+            </thead>
+
+            <tbody>
+              <row>
+                <entry>Virtual network </entry>
+
+                <entry>terraform-azurerm-hpcc/modules/virtual_network </entry>
+              </row>
+
+              <row>
+                <entry>Storage account </entry>
+
+                <entry>terraform-azurerm-hpcc/modules/storage_account </entry>
+              </row>
+
+              <row>
+                <entry>AKS </entry>
+
+                <entry>terraform-azurerm-hpcc </entry>
+              </row>
+            </tbody>
+          </tgroup>
+        </informaltable>
+
+        <para>After you clone the terraform-azurerm-hpcc repository you will
```

Comment:
S/B Present tense: After you clone the terraform-azurerm-hpcc repository you have access to the modules in that repository.